pynecone/components/tags/tag.py

```diff
@@ -209,3 +211,23 @@ def is_valid_prop(prop: Optional[Var]) -> bool:
             Whether the prop is valid.
         """
         return prop is not None and not (isinstance(prop, dict) and len(prop) == 0)
+
+    @staticmethod
+    def format_style(prop: dict) -> str:
+        """Format style.
+
+        Args:
+            prop: The style.
+
+        Returns:
+            The formatted style.
+        """
+        ret = utils.json_dumps(prop)
+
+        # This substitution is necessary to unwrap var values.
+        if ret.find('"{') > 0:
+            ret = re.sub('"{', "", ret)
+            ret = re.sub('}"', "", ret)
+            ret = re.sub('\\\\"', '"', ret)
```

Comment:
I remember this specific sub was causing errors for something a while back, do you remember the specific case causing this @picklelo, I'll test it

Reply:
I think this sub was necessary for fixing another issue with indexing state vars i.e. `prop=State.dict["key"]` was getting formatted wrong without this. I added unit tests for that though, so if they are passing we should be good.

Reply:
Oh but now they may not be valid within style props. We need to make sure we can pass in the state indexing a string like above^ for style props also.

Reply:
@picklelo 

> Oh but now they may not be valid within style props. We need to make sure we can pass in the state indexing a string like above^ for style props also.

How to make sure? Is there a case to check it?